src/cow-react/modules/swap/helpers/getSwapButtonState.ts

```diff
@@ -74,7 +74,7 @@ export function getSwapButtonState(input: SwapButtonStateParams): SwapButtonStat
   const isValid = !input.inputError && input.feeWarningAccepted && input.impactWarningAccepted
   const swapBlankState = !input.inputError && !input.trade
 
-  if (quoteError) {
+  if (quoteError && ![WrapType.WRAP, WrapType.UNWRAP].includes(wrapType)) {
```

Comment:
ubernit, ignore since is merged. But this either calls for a function, or a const for readability. If this is used somewhere else, maybe the function is a good idea